manifests/agent/install.pp

```diff
@@ -1,8 +1,15 @@
 # Install the puppet client installation
 class puppet::agent::install {
+  case $::osfamily {
+    Windows : {
+      package { $puppet::params::client_package:
+        ensure   => $::puppet::version,
+        provider => $::puppet::params::win_provider,
+      }
+    }
 
-  package { $puppet::params::client_package:
-    ensure => $::puppet::version,
+    default : {
+      package { $puppet::params::client_package: ensure => $::puppet::version, }
```

Comment:
Dang Geppetto... that is something I feel is broken about their formatting... I agree this needs changing back, I just overlooked it.